test/seldon/model/test-models

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+COMPUTE_NAMESPACE=seldon-regression-test
+
+EXPECTED_IRIS_RESULT_PAYLOAD='{"data":{"names":["t:0","t:1","t:2"],"ndarray":[[0.0006985194531162835,0.00366803903943666,0.995633441507447]]},"meta":{"requestPath":{"classifier":"seldonio/sklearnserver:1.14.1"}}}'
+EXPECTED_MOCK_RESULT_PAYLOAD='{"data":{"names":["proba"],"ndarray":[[0.43782349911420193]]},"meta":{"requestPath":{"classifier":"seldonio/mock_classifier:1.7.0-dev"}}}'
+
+echo ">>> Testing iris model..."
+IRIS_RESULT_PAYLOAD=$(kubectl -n $COMPUTE_NAMESPACE exec deployments/iris-model-default-0-classifier -c classifier -- \
+    curl -X POST http://localhost:8000/api/v1.0/predictions \
+    -H 'Content-Type: application/json' -d '{"data": {"ndarray": [[1,2,3,4]] }}')
+if [ "${EXPECTED_IRIS_RESULT_PAYLOAD}" != "${IRIS_RESULT_PAYLOAD}" ]; then
+  echo "UNEXPECTED IRIS MODEL RESULT: ${IRIS_RESULT_PAYLOAD}"
+  exit 1
+fi
+
+echo ">>> Testing mock classifier model..."
+MOCK_RESULT_PAYLOAD=$(kubectl -n $COMPUTE_NAMESPACE exec deployments/mock-classifier-model-0-classifier -c classifier -- \
+    curl -X POST http://localhost:8000/api/v1.0/predictions \
+    -H 'Content-Type: application/json' \
+    -d '{"data": {"ndarray": [[1.0, 2.0, 5.0]] }}')
+if [ "${EXPECTED_MOCK_RESULT_PAYLOAD}" != "${MOCK_RESULT_PAYLOAD}" ]; then
+  echo "UNEXPECTED MOCK MODEL RESULT: ${MOCK_RESULT_PAYLOAD}"
+  exit 1
+fi
```

Comment:
please configure your editor to always add a new line at the end

Reply:
This is my fault, I did change the setting, but perhaps this file got in before the change, ill update it